core/docker-entrypoint.sh

```diff
@@ -27,14 +27,15 @@ function wait_for_mysql () {
 echo "# This file is autogenerated at container startup." > /etc/mailman.cfg
 
 # Check if $MM_HOSTNAME is set, if not, set it to the value returned by
-# `hostname -i` command to set it to whatever IP address is assigned to the
+# `hostname` command to set it to whatever hostname is assigned to the
 # container.
 if [[ ! -v MM_HOSTNAME ]]; then
-	export MM_HOSTNAME=`hostname -i`
+	export MM_HOSTNAME=`hostname`
 fi
 
+# SMTP_HOST defaults to the gateway
 if [[ ! -v SMTP_HOST ]]; then
-	export SMTP_HOST='172.19.199.1'
+	export SMTP_HOST=$(/sbin/ip route | awk '/default/ { print $3 }')
```

Comment:
Can you print parsed value this here too?

It can be a very useful debugging information from `docker logs` of the container startup.

Something like:

```
echo "SMPT_HOST not specified, using the gateway as default $SMTP_HOST"
```

Might want to check my bash above ;-) I think env vars inside `echo ""` prints value, right?

Reply:
Done.